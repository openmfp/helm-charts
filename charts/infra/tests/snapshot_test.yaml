suite: snapshot
values:
  - ../test-values.yaml
tests:
  - it: matches the snapshot
    asserts:
      - matchSnapshot: {}
  - it: disables externalsecrets
    set:
      externalSecrets:
        accountOperatorSaKubeconfig: null
    asserts:
      - matchSnapshot: {}
  - it: with kcp
    set:
      kcp:
        enabled: true
      istio:
        passThrough:
          gateway:
            enabled: true
            hosts:
              - "passthrough.my-domain.com"
            port: 8000
            name: pass-https
            protocol: HTTPS

    asserts:
      - matchSnapshot: {}

