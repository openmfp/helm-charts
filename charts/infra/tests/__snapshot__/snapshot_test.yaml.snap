matches the snapshot:
  1: |
    apiVersion: networking.istio.io/v1
    kind: Gateway
    metadata:
      name: gateway
      namespace: NAMESPACE
    spec:
      selector:
        istio: gateway
      servers:
        - hosts:
            - '*'
          port:
            name: http
            number: 8080
            protocol: HTTP
  2: |
    apiVersion: networking.istio.io/v1
    kind: ServiceEntry
    metadata:
      name: openmfp-https
      namespace: NAMESPACE
    spec:
      hosts:
        - example.com
      location: MESH_EXTERNAL
      ports:
        - name: https
          number: 443
          protocol: TLS
      resolution: DNS
